src/components/NavBar/index.tsx

```diff
@@ -60,13 +60,9 @@ const NeracoosNavBar = () => {
             <NavLink href={paths.home}>Home</NavLink>
 
             <RegionDropdown closeParent={close} />
-
-            {!isMariners ? (
-              <NavItem>
-                <NavLink href={paths.models}>Model Viewer</NavLink>
-              </NavItem>
-            ) : null}
-
+            <NavItem>
+              <NavLink href={paths.models}>Model Viewer</NavLink>
+            </NavItem>
```

Comment:
Right now the model viewer (and maybe some of the servers behind it) aren't quite ready for full public consumption, so we still want to check if the site is at mariners.neracoos.org and hide it from the menu if that is the case. 

Probably the better route (so we avoid a flash as things get rehydrated and update), would be to not show it by default, and only show it if `!window.location.href.includes("://mariners.neracoos.org")`.